gems/quilt_rails/lib/generators/quilt/demo_app/demo_app_generator.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Quilt
+  class DemoAppGenerator < Rails::Generators::Base
+    source_root File.expand_path('templates', __dir__)
+    desc "This generator adds a demo React app with all the quilt toolings."
+
+    def install_js_dependencies
+      say "Installing application js dependencies"
+      system("yarn add "\
+        "@shopify/react-router "\
+        "@shopify/polaris "\
+        "@shopify/react-async "\
+        "@shopify/react-network "\
+        "@shopify/react-performance "\
+        "@shopify/react-i18n "\
+        "@shopify/react-i18n-universal-provider") unless Rails.env.test?
+    end
+
+    def add_path_sk_config
+      file_path = "config/sewing-kit.config.ts"
+
+      inject_into_file(
+        file_path,
+        File.read(File.expand_path(find_in_source_paths('path_import.ts'))),
+        after: "import {Plugins} from '@shopify/sewing-kit';"
+      )
+    end
+
+    def add_polaris_sk_config
+      file_path = "config/sewing-kit.config.ts"
+
+      gsub_file(
+        file_path,
+        "plugins: [", "plugins: [#{File.read(File.expand_path(find_in_source_paths('polaris_sass_plugin.ts')))}"
+      )
+    end
```

Comment:
a bit of a flaky solution

Reply:
If the config file was JSON it would be more language agnostic. As it stands you'll likely have to resort to pattern matching in some way to do this.